terraform/paas/route.tf

```diff
@@ -1,7 +1,7 @@
 data "cloudfoundry_route" "app_route_internet" {
-  count    = var.additional_routes
   domain   = data.cloudfoundry_domain.internet.id
-  hostname = var.paas_additional_route_name
+  for_each = toset(var.paas_additional_route_names)
```

Comment:
Nice
By convention the `for_each` line comes first

Reply:
done